pkg/NAMESPACE

```diff
@@ -97,3 +103,5 @@ export("jsonFile",
        "uncacheTable")
 
 export("sparkRSQL.init")
+export("print.structType",
```

Comment:
As far as I know, if you want the print methods to trigger on those environments, you have to export the print method even if you don't export the actual wrapper function.